components/HomeScreen.jsx

```diff
@@ -3,11 +3,11 @@ import Articles from './sub-components/Articles';
 import NavBar from "./sub-components/NavBar";
 import Header from "./sub-components/Header";
 
-const HomeScreen = ({setVotes, loading, setLoading}) => {
+const HomeScreen = ({setVotes, isLoading, setIsLoading}) => {
```

Comment:
Why would this be sent from app, why does app.jsx even have this, what purpose does it serve?